ghc.mk

```diff
@@ -1091,7 +1091,7 @@ $(eval $(call bindist-list,.,\
     $(wildcard libraries/*/*/dist-install/doc/) \
     $(filter-out settings llvm-targets,$(INSTALL_LIBS)) \
     $(RTS_INSTALL_LIBS) \
-    $(filter-out %/project.mk mk/config.mk %/mk/install.mk,$(MAKEFILE_LIST)) \
+    $(filter-out testsuite/utils/check-ppr/% testsuite/utils/check-api-annotations/% %/project.mk mk/config.mk %/mk/install.mk,$(MAKEFILE_LIST)) \
```

Comment:
Presumably this shouldn't be here?

Reply:
Not anymore: the revert in d9b6015d1942aa176e85bb71f34200bab54e1c9c makes it unnecessary, will remove this commit.